data/pathfinder/paizo/player_companion/adventurers_armory/aa_abilities.lst

```diff
@@ -0,0 +1,23 @@
+# CVS $Revision: 12957 $ $Author: jbaril $ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
+SOURCELONG:Adventurer's Armory	SOURCESHORT:AA	SOURCEWEB:http://paizo.com/products/btpy8dmf?Pathfinder-Player-Companion-Adventurers-Armory	SOURCEDATE:2010-04
+# Original Entry by: Stefan Radermacher
```

Comment:
He did most of this source. All I did was get the one piece he didn't do... I CAN add my name, but honestly, I wasn't worried about it.